src/components/Staking/ShanghaiCapella.tsx

```diff
@@ -0,0 +1,97 @@
+// Import libraries
+import React, { FC } from "react"
+import {
+  Flex,
+  Text,
+  UnorderedList,
+  ListItem,
+  FlexProps,
+} from "@chakra-ui/react"
+// Components
+import ButtonLink from "../ButtonLink"
+import Card from "../Card"
+
+interface IProps extends FlexProps {}
+const ShanghaiCapella: FC<IProps> = (props) => {
+  const upgrades = [
+    {
+      heading: "Shanghai",
+      emoji: "📜",
+      content: (
+        <>
+          <Text>
+            Push withdrawals will be enabled in the upcoming Shanghai upgrade
```

Comment:
What are "push withdrawals" vs. regular withdrawals? 

I believe they're the same. Should we use the term "push withdrawals" or will that just potentially confuse?

Reply:
More of a remnant from using that term heavily in the original copy... since we've reduced its usage, I don't think it's worth using it here anymore.